CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bug fixes
 
 * [#184](https://github.com/rubocop-hq/rubocop-rails/issues/184): Fix `Rake/Environment` to allow task with no block. ([@hanachin][])
+* [#187](https://github.com/rubocop-hq/rubocop-rails/pull/187): Fix the issue that excluded files in rubocop-rails did not work. ([@sinsoku][])
```

Comment:
I think it's `an issue` instead of `the issue`.

Reply:
Thanks for your review.
I fixed it.